easybuild/easyconfigs/g/Gaussian/Gaussian-09.e.01-AVX.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
+
+name = 'Gaussian'
+version = '09.e.01'
+versionsuffix = '-AVX'
+
+homepage = 'https://www.gaussian.com/'
+
+# For the sake of users I advise changing the last line in the description
+# and versionsuffix to match the actual installed version.
+description = """
+Gaussian provides state-of-the-art capabilities for electronic structure
+modeling. Gaussian 09 is licensed for a wide variety of computer
+systems. All versions of Gaussian 09 contain every scientific/modeling
+feature, and none imposes any artificial limitations on calculations
+other than your computing resources and patience.
+
+This is the official gaussian AVX build.
+"""
+
+toolchain = SYSTEM
+
+# Gaussian is licensed software.
+# You need to pick the tar file of your choice from the DVD,
+# put it in your sources tree (should be a tree that ordinary users can't read)
+# and change the name in "sources"
+sources = ['09.e.01-AVX-A64-114X.tgz']
```

Comment:
@akesandgren Please use `%(version)s` here to avoid surprises when `version = '...'` is tweaked?

Same goes for `%(versionsuffix)s`

Reply:
Done